.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,34 @@
+Name: Clean up abandoned files
+
+on:
+  schedule:
+    - cron: '30 0 * * 1'
+  push:
+    branches:
+    - master
+jobs:
+  update:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout webref
+      uses: actions/checkout@master
+      with:
+        path: webref
+    - name: Setup node.js
+      uses: actions/setup-node@master
+      with:
+        node-version: 10.x
+    - name: Run clean up
+      run: node tools/clean-abandonned-files.js
+    - uses: peter-evans/create-pull-request@v2
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        title: Remove files no longer referenced
+        commit-message: "Remove files no longer referenced"
+        body: |
+          This removes previously extracted files that are no longer referenced in browser-specs
+        assignees: tidoust, dontcallmedom
+        draft: true
+        branch: cleanup
+        branch-suffix: timestamp
```

Comment:
If I understand the action correctly, the `branch-suffix` setting will create a new PR each time, instead of reusing an existing one. That means one new PR created every 6 hours until we merge the changes...

"Default" behavior (when `branch-suffix` is not defined) seems preferable:
https://github.com/peter-evans/create-pull-request#action-behaviour
(not sure if `delete-branch` should be set)

Reply:
it will only create a pull request if there are changes, which I think would be OK if we use the cron job (leaving us a week to react), but you're right it wouldn't be if we use on-push.

(using the default behavior won't work - it means once we close the first pull request the script generates, it won't ever be opened again IIUC)